LayoutTests/svg/dom/svganimatedlength-initial-values-expected.txt

```diff
@@ -0,0 +1,197 @@
+CONSOLE MESSAGE: Error: Invalid value for <circle> attribute cx="foobar"
```

Comment:
Is it good to log these in the console? I guess maybe it's helpful for website developers?